### PR TITLE
Create mdcb-2.4.md

### PR DESCRIPTION
**Pull request by carolwkc**
### For internal users - please add a Jira DX PR ticket to the subject!
---
### Description

Release Notes for MDCB

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/release-notes/mdcb-2.3.md [hunk 1]
```diff
@@ -6,6 +6,16 @@ menu:
...
 ---
 
+## 2.3.1
+
+Release date: 2023-08-31
+
+### Fixed
+
+- In MDCB 2.3, the embedded OAS API Definition introduced in 5.1 is not backward compatible. It causes Gateway panic when MDCB is connecting to Tyk 5.0.x or earlier releases. In this fix, MDCB will transform the old API Definition to new format to avoid panic.
+- User should use URL-encoded values in username and password of a MongoDB connection string if it contains following characters - "?", "@". The same connection string should always be accepted by both mgo and mongo-go drivers. (Note: Same fix for Dashboard will be available in upcoming release Tyk Dashboard v5.0.6 and v5.2.0)
```
Comment (dcs3spp) [inline tyk-docs/content/release-notes/mdcb-2.3.md]:
```suggestion
- Users should use URL-encoded values in username and password of a MongoDB connection string if it contains following characters - "?", "@". The same connection string should always be accepted by both mgo and mongo-go drivers. (Note: Same fix for Dashboard will be available in upcoming release Tyk Dashboard v5.0.6 and v5.2.0)
```

File: tyk-docs/content/release-notes/mdcb-2.4.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+## 2.4.0
+
+Release date: 2023-09-14
+
+MDCB 2.4.0 is an update for compatibility for synchronisation with Tyk v5.2 API Definitions. It also enables gateway information visualisation on Tyk Dashboard v5.2+
+
+### Added
+- Track number of connected gateways and gateway info. The connection statistics can be queried from Tyk Dashboard v5.2+. This allow greater visibility for Operation teams on number of gateways they are using
```
Comment (dcs3spp) [inline tyk-docs/content/release-notes/mdcb-2.4.md]:
```suggestion
- Track number of connected gateways and gateway info. The connection statistics can be queried from Tyk Dashboard v5.2+. This allow greater visibility for Operation teams on number of gateways they are using.
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks for submitting PR Carol, 

PR lgtm @caroltyk

Made minor suggestions and then we can release

**Comment by carolwkc** [comment]
Thanks @dcs3spp , the changes are applied.

**Review by kolavcic — APPROVED** [review]
LGTM

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**